cmd/clusterctl/definition/ibmcloud/machines.yaml.template

```diff
@@ -0,0 +1,44 @@
+items:
+- apiVersion: "cluster.k8s.io/v1alpha1"
+  kind: Machine
+  metadata:
+    generateName: ibmcloud-master-
+    labels:
+      set: master
+  spec:
+    providerSpec:
+      value:
+        apiVersion: "ibmcloudproviderconfig/v1alpha1"
+        kind: "IbmcloudMachineProviderSpec"
+        hostname: <host name>
+        domain: <domain name>
```

Comment:
@jichenjc do we need floating IP here for external access?

Reply:
looks like (from my hands on) the public IP is assigned by default and you can't request one machine with given IP address ,so no need to put into template but we can get ip later when machine created

Reply:
good to know, thanks ;)